reference/conanfile/tools/gnu/autotools.rst

```diff
@@ -106,7 +106,7 @@ This is just an "alias" of ``self.make(target="install")``
 
 Parameters:
     - **args** (Optional, Defaulted to ``None``): List of arguments to use for the
-      ``make install`` call. By default an argument ``DESTDIR=self.package_folder`` is added to the
+      ``make install`` call. By default an argument ``DESTDIR=unix_path(self, self.package_folder)`` is added to the
```

Comment:
I think we should add a link to the tool as it's not that evident what this does

Reply:
Oh, absolutely!